test/unit/tracks/tracks.test.js

```diff
@@ -308,8 +308,8 @@ test('when switching techs, we should not get a new text track', function() {
   ok(htmltracks === flashtracks, 'the tracks are equal');
 });
 
-if (Html5.supportsNativeTextTracks()) {
-  test('listen to remove and add track events in native text tracks', function(assert) {
+if (Html5.supportsNativeTextTracks() && ('onremovetrack' in Html5.TEST_VID.textTracks)) {
```

Comment:
Should we just add this `onremovetrack` check to the supportsNativeTextTracks check?